solution/ui/regulations/eregs-component-lib/package.json

```diff
@@ -14,6 +14,7 @@
     "chokidar": "^3.5.3",
     "localforage": "^1.10.0",
     "lodash": "^4.17.21",
+    "mitt": "^3.0.1",
```

Comment:
mitt eventbus library replaces the deprecated Vue 2 event emitter interface

https://v3-migration.vuejs.org/breaking-changes/events-api.html